### DISCUSSION TITLE
ingressing hcpya data

### DISCUSSION THREAD

**Discussion by bogpetre** [PennLINC/qsiprep#732] [Ideas]
Hi

I noticed that there are plans to add support for ingressing hcpya data from the HCP minimally preprocessed data folder structures (rather than BIDS), but that there's no implementation yet for this. I'm interested in potentially using HCP preprocessed data and am trying to evaluate whether or not it's worthwhile to try to implement this ingressing functionality myself. I'm wondering if any work has been done on this yet, if there are an key difficulties you may have encountered that impeded its implementation thus far, or if there's ongoing work I should be aware of. Based on what I can see from the code base, it looks like nothing HCP specific has been implemented yet.

**Comment by arokem** [comment]
Hey @bogpetre : there's also [functionality in DIPY](https://github.com/dipy/dipy/blob/master/dipy/data/fetcher.py#L2267) to download processed HCP diffusion data from AWS and organize it in a (I believe) BIDS-compatible fashion.

**Comment by mattcieslak** [comment]
To run the recon workflows it would need to look like qsiprep derivatives, which will be a bit different from raw BIDS. @bogpetre if you're up for trying out a PR it won't be much different from the UKB workflow and I'd be happy to help

**Comment by mattcieslak** [comment]
This exists now! Please see https://github.com/pennlinc/ingress2qsirecon
